src/main/java/vswe/stevesfactory/interfaces/ContainerManager.java

```diff
@@ -30,6 +33,19 @@ public boolean canInteractWith(EntityPlayer entityplayer) {
 
     @Override
     public void detectAndSendChanges() {
+        try {
+            detectAndSendChanges0();
```

Comment:
`detectAndSendChangesInternal` maybe?

Reply:
0 suffix is sort of a established (though not as wide spread) convention in java. In any case, I prefer Impl suffix.

Reply:
oh good to know 👍